geode-connectors/src/main/java/org/apache/geode/connectors/util/internal/MappingConstants.java

```diff
@@ -27,5 +27,8 @@
   public static final String GROUP_NAME = "groups";
   public static final String PDX_CLASS_FILE = "pdx-class-file";
 
+  public static final String NOT_TO_CREATE_MAPPING_ON_PROXY =
+      "Not to create jdbc-mapping on proxy region.";
```

Comment:
If I saw this message in the describe mapping output I would not understand what it is telling me. I'm also not able to understand what you are doing in the implementation that uses this string so I'm not able to suggest a better string.